test/AuthenticationController.spec.ts

```diff
@@ -237,13 +237,13 @@ function mockDependenciesForAuth(container: Container, verifies: boolean, sessio
     if(verifies) {
         const signatures: SessionSignature[] = [
             {
-                address: ALICE,
+                address: validAccountId(ALICE).address,
```

Comment:
Another approach would be to convert the value of `ALICE` using Logion's prefix.